pep-0675.rst

```diff
@@ -558,14 +564,10 @@ Runtime behavior
 
 This PEP does not change the runtime behavior of ``Literal``.
 
-Backwards compatibility
-=======================
-
 Backwards compatibility: ``Literal[str]`` is acceptable at runtime, so
 this doesn't require any changes to the Python runtime itself. :pep:`586`
 already backports ``Literal``, so this PEP does not need to change it.
 
```

Comment:
```suggestion
Backwards Compatibility
=======================

``Literal[str]`` is acceptable at runtime, so
this doesn't require any changes to the Python runtime itself. :pep:`586`
already backports ``Literal``, so this PEP does not need to change it.


```

Per PEP 12 (PEP-0012), this is a standard top-level section heading, and two blank lines are nominally required before such. If you view it as redundant with the `Runtime behavior` [sic, might wanna fix that inconsistent case :)] section, you could just elide the latter and inline the one line of text there into this section.

Reply:
Done! There were two sections of "Backwards Compatibility", which is why I'd made the earlier change. I've merged them.

Reply:
Gotcha, thanks! I was wondering about that, but I figured you had your reasons to separate the type-checking and runtime compat aspects so I didn't want to second-guess that.